_sass/_components/layout.scss

```diff
@@ -48,11 +48,51 @@
     color: $color-white;
   }
 
-  a {
+  a
+   {
     color: $color-bright;
 
     &:hover {
       color: $color-bright-hover;
     }
   }
+  .usa-button,
+  .usa-button-primary,
+  .usa-button:visited,
+  .usa-button-primary:visited,
+  button,
```

Comment:
It's a button style but for `.background-dark` layout type, so I'm not sure if it should be moved. It's nice to see all the changes for that layout together.